src/main/java/com/example/nlern/models/FullTest.java

```diff
@@ -0,0 +1,14 @@
+package com.example.nlern.models;
+
+import lombok.Getter;
+import lombok.Setter;
+import java.util.List;
+
+@Getter @Setter
+public class FullTest {
+    Test test;
+    List<Question> questions;
+    List<Integer> deleted;
```

Comment:
Ids, or what exactly do you mean by deleted?

Reply:
Yes. Corrected